hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/submerkle/EntityId.java

```diff
@@ -271,6 +271,14 @@ public AccountID toGrpcAccountId() {
                 .build();
     }
 
+    public com.hedera.hapi.node.base.AccountID toPbjAccountId() {
```

Comment:
null annotation missing

Reply:
Question: Should we already deprecate all converter methods?

Reply:
It would be a good idea to deprecate for removal the PbjConverter class right after this PR, so we get compiler warnings to replace that with proper conversions as we build out the module implementations.